addons/web_editor/static/src/js/backend/convert_inline.js

```diff
@@ -1166,6 +1117,65 @@ function _applyColspan(element, colspan, tableWidth) {
     const width = Math.round(tableWidth * widthPercentage * 100) / 100;
     element.style.setProperty('max-width', width + 'px');
 }
+/**
+ * Take an element with a background image and return a string containing the
+ * VML code to display the same image properly in Outlook, with its contents
+ * inside.
+ * TODO: We assume:
+ *      - background-size: cover,
+ *      - background-repeat: no-repeat,
+ *      - size 100%
+ *      - content is centered x/y
+ * These assumptions may not hold for cover snippet and other uses. This needs
+ * to account for that.
+ * TODO: centering span probably not needed with `v-text-anchor:middle` present.
+ *
+ * @param {Element} backgroundImage
+ * @returns {string}
+ */
+function _backgroundImageToVml(backgroundImage) {
+    // Create the outer structure.
+    const clone = backgroundImage.cloneNode(true);
+    const div = document.createElement('div');
+    div.replaceChildren(...clone.childNodes);
+    [['fontSize', 0], ['height', '100%'], ['width', '100%']].forEach(([k, v]) => div.style[k] = v);
+    const vmlContent = document.createElement('div');
+    vmlContent.append(div);
+
+    // Preserve important inherited properties without ancestor context.
+    const style = getComputedStyle(backgroundImage);
+    for (const prop of ['color', 'font-size', 'font-family', 'font-weight', 'font-style', 'text-decoration', 'text-transform']) {
+        div.style[prop] = backgroundImage.style[prop] || style[prop];
+    }
+    [...div.children].forEach(child => child.style.setProperty('font-size', child.style.fontSize || style.fontSize));
+
+    // Prepare the top element for hosting the VML image.
+    for (const prop of ['background', 'background-image', 'background-repeat', 'background-size']) {
+        clone.style.removeProperty(prop);
+    }
+    clone.style.padding = 0;
+    clone.className = clone.className.replace(/p[bt]\d+/g, ''); // Remove padding classes.
+    const url = backgroundImage.style.backgroundImage.match(/url\("?(.+?)"?\)/)[1];
+    clone.setAttribute('background', url);
```

Comment:
Not sure this is actually needed. [backgrounds.cm](https://backgrounds.cm) does it but [MJML](https://github.com/mjmlio/mjml/blob/7c8ee3fe1f2c320a7dd55b7688f4daeb6e0c0d6b/packages/mjml-section/src/index.js#L383) doesn't. I don't really see a reason for it since we already apply the background with VML.

For reference: https://www.caniemail.com/search/?s=background